sim/scripts/synth_fpga.tcl

```diff
@@ -0,0 +1,132 @@
+# See LICENSE for license details.
+
+
+# Amazon FPGA Hardware Development Kit
+#
+# Copyright 2016 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Amazon Software License (the "License"). You may not use
+# this file except in compliance with the License. A copy of the License is
+# located at
+#
+#    http://aws.amazon.com/asl/
+#
+# or in the "license" file accompanying this file. This file is distributed on
+# an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, express or
+# implied. See the License for the specific language governing permissions and
+# limitations under the License.
```

Comment:
Is this file being copied from aws-fpga into firesim?  Why?  It would be helpful to have a file-level comment  explaination of why it is being vendored into firesim and what this copy does that the original didn't do.

Reply:
Added another comment explaining the purpose of this file.